src/Microsoft.AspNet.Http.Extensions/HeaderDictionaryExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static void AppendCommaSeparatedValues(this IHeaderDictionary headers, st
         /// <returns>the associated values from the collection separated into individual values, or StringValues.Empty if the key is not present.</returns>
         public static string[] GetCommaSeparatedValues(this IHeaderDictionary headers, string key)
         {
-            return ParsingHelpers.GetHeaderSplit(headers, key);
+            return ParsingHelpers.GetHeaderSplit(headers, key).ToArray();
```

Comment:
This actually caused a subtle behavior change. The implicit converter would return null in some cases, where ToArray always returns at least an empty value. The same for ToString.
https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.Primitives/StringValues.cs#L104
https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.Primitives/StringValues.cs#L85